### DISCUSSION TITLE
How to configure WG-EASY ? (allowed ip)

### DISCUSSION THREAD

**Discussion by williams407** [wg-easy/wg-easy#228] [Q&A]
Hello, i'm totally new on docker, i never used...
I can't acces to my device on lan, i think i need to change allowed ip

How can i do this ?

Thanks!

**Comment by eandersons** [comment]
It would help to see how you launched WireGuard Easy container - content of `docker-compose.yml` or the full `docker run` command.

All configuration environment variables are described in the section ["Options"](/WeeJeWel/wg-easy#options).

**Reply by williams407** [reply]
As a mentionned early, i have no idea how to use "docker run" or "docker-compose.yml" because i never using docker or linux and this is why i ask for help, when i try using "docker run" its say ""docker run" requires at least 1 argument. See 'docker run --help'.

Thanks for your time

**Reply by eandersons** [reply]
Did you carefully read the instructions? `README.md`  (especially the section [2. Run WireGuard Easy](/WeeJeWel/wg-easy#2-run-wireguard-easy)) contains all the necessary information (including the full `docker run` command) to successfully run WireGuard Easy.

Another option is to copy or download [`docker-compose.yml`](/WeeJeWel/wg-easy/blob/master/docker-compose.yml) and start the WireBuard Easy container with the command `docker-compose up --detach` ([Docker Compose](https://docs.docker.com/compose/install/) is required for this).

**Reply by williams407** [reply]
Hello,
I finally succesfully run docker-compose.yml, sorry mate i'm totally noob on Linux.
Its work but i can't accès to my lan :( 

What i can do for that ?
My lan is configured in 192.168.1.1 sub 255.255.255.0
Thanks for your patience

**Reply by eandersons** [reply]
What exactly do you mean by "i can't accès to my lan"?
You cannot connect to WireGuard VPN?
You can connect to WireGuard VPN but cannot access internet?
You cannot connect to other machines on your LAN from a device that is connected to WireGuard VPN?

It would help to help you if you provided more information. For example, what are you trying to achieve, what did you try, what errors did you encounter.

**Reply by kuznetsov-as** [reply]
@eandersons 
Greetings, please tell me how to connect to other machines on LAN from a device that is connected to WireGuard VPN? 

I am trying to follow this instruction:
https://iliasa.eu/wireguard-how-to-access-a-peers-local-network/

It's very simple and I follow it, but when I use it with wg-easy nothing works.
Also I don't understand how to change wg0 in the context of wg-easy - after reboot, my changes to the wg0 are gone.
